zerver/webhooks/uptimerobot/doc.md

```diff
@@ -14,6 +14,8 @@ Receive Zulip notifications from UptimeRobot!
     notifications sent to. If you do not specify a `stream`, the bot will
     send notifications via PMs to the creator of the bot.
 
+1. {!event-filtering-instruction.md!}
```

Comment:
Since this isn't a separate bullet point on the other integration pages, I would just indent this so that it renders as part of the above section and remove the number ("1.").